variables.tf

```diff
@@ -209,26 +477,26 @@ variable "addon_virtual_network_id" {
   description = "The name of the addon vnet link vnet id"
 }
 
-# app insights
-variable "application_insights_enabled" {
-  description = "Specify the Application Insights use for this App Service"
-  default     = true
-}
+# # app insights
```

Comment:
If this is not required, can you please remove this, same is for others (below one).